tests/views/test_message.py

```diff
@@ -1612,6 +1615,128 @@ def test_conversation_reply_no_edit_permission(self, mock_request, mock_get_jwt)
         self.assertNotIn("Close conversation".encode(), response.data)
         self.assertNotIn("Send message".encode(), response.data)
 
+    @requests_mock.mock()
+    @patch("response_operations_ui.controllers.message_controllers._get_jwt")
+    @patch("flask_login.utils._get_user")
+    def test_flash_redirect_to_technical_inbox(self, mock_request, current_user, mock_get_jwt):
+        with self.client.session_transaction() as session:
+            session["messages_survey_selection"] = "technical"
+        with self.client.session_transaction() as session:
+            session["user_id"] = "test-id"
+        mock_get_jwt.return_value = "blah"
```

Comment:
Not a fan of this, although I realised you have copied it

Reply:
I had meant to update them but forgot. This has now been done.